spec/unit/os_inspector_spec.rb

```diff
@@ -67,6 +67,17 @@
     end
   end
 
+  describe "#get_os_from_redhat_release" do
+    it "gets os info from redhat release file" do
+      FakeFS::FileSystem.clone("spec/data/os/rhel6/etc/redhat-release",
+        "/etc/redhat-release")
+      os = inspector.send(:get_os_from_redhat_release, system)
+
+      expect(os.name).to eq "Red Hat Enterprise Linux Server"
+      expect(os.version).to eq "6.5 (Santiago)"
+    end
+  end
+
```

Comment:
Cornelius asked for a test for this method. See:
https://github.com/SUSE/machinery/pull/522/files#r22998839